tracer/src/Datadog.Trace.Tools.Runner/RunCiCommand.cs

```diff
@@ -3,23 +3,285 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent.DiscoveryService;
+using Datadog.Trace.Ci;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
 using Spectre.Console;
 using Spectre.Console.Cli;
 
 namespace Datadog.Trace.Tools.Runner
 {
-    internal class RunCiCommand : Command<RunCiSettings>
+    internal class RunCiCommand : AsyncCommand<RunCiSettings>
     {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(RunCiCommand));
+        private readonly ApplicationContext _applicationContext;
```

Comment:
I thought the use of underscore marking for members was discouraged in c#

Reply:
> nit: I thought the use of underscore marking for members was discouraged in c#

No, the opposite, it's mandated by style cop 😉